ckanext/geocat/tests/test_distribution_metadata.py

```diff
@@ -35,81 +36,78 @@ def _load_xml(self, filename):
     def _is_multi_lang(self, value):
         for lang in ['de', 'fr', 'it', 'en']:
             self.assertIn(lang, value)
-        
-    def test_fields(self):
+
+    def test_all_resources(self):
         xml = self._load_xml('complete.xml')
-        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier)
+        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier_deprecated_protocols)
         distributions = dataset.get('resources')
-
         self.assertEquals(4, len(distributions))
 
-        fields = [
-            'title',
-            'description',
-            'issued',
-            'modified',
-            'language',
-            'url',
-            'download_url',
-            'media_type',
-            'rights',
-            'format',
-        ]
-        for dist in distributions:
-            for field in fields:
-                self.assertIn(field, dist)
-
-            for key, value in dist.iteritems():
-                self.assertIn(key, fields)
-
-            # check multilang fields
-            self._is_multi_lang(dist.get('title'))
-            self._is_multi_lang(dist.get('description'))
-
-    def test_fields_values(self):
+    def test_deprecated_protocol_for_WWW_DOWNLOAD(self):
         xml = self._load_xml('complete.xml')
-        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier)
+        dataset = self.csw_map.get_metadata(xml, self.geocat_identifier_deprecated_protocols)
```

Comment:
can you use `geocat-testdata.xml` with geocat_identifier_deprecated_protocols? 
so that for all tests the same xml was used?

Reply:
No, `geocat-testdata.xml` should contain only valid protocols and testdata so that it matches what we can expect on the platform currently.